src/test/java/myfirstproject/service/impl/WriteFileImplTest.java

```diff
@@ -0,0 +1,30 @@
+package myfirstproject.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import myfirstproject.model.Fruit;
+import myfirstproject.service.WriteFile;
+import org.junit.Test;
+
+public class WriteFileImplTest {
+    private static final String PATH_TO_NEW_FILE = "src/test/resources/reportFile.csv";
+    private static final String WRONG_PATH = "src/resources/reportFile.csv";
+    private static final WriteFile WRITE_FILE = new WriteFileImpl();
+
+    @Test(expected = RuntimeException.class)
+    public void readWrongFile_not_Ok() {
+        WRITE_FILE.writeToFile(WRONG_PATH, Collections.EMPTY_MAP);
+    }
+
+    @Test
+    public void isCreatedFileExist_Ok() {
+        final File file = new File(PATH_TO_NEW_FILE);
+        Map<Fruit,Integer> testMap = new HashMap<>();
+        WRITE_FILE.writeToFile(PATH_TO_NEW_FILE, testMap);
+        assertTrue(file.exists());
+    }
```

Comment:
isn't enough for testing
let's also check content of the file (so map shouldn't be empty)